tests/flow/random_graph.py

```
@@ -0,0 +1,262 @@
+from index_utils import *
```

Ruff (0.12.2)

`from index_utils import *` used; unable to detect undefined names

(F403)